### ISSUE TITLE
refine cluster using polygon

### ISSUE THREAD

**Issue by agbondy** [JaneliaSciComp/JRCLUST#115]
In the feature view, is it possible to draw a polygon for the purposes of refining a cluster -- that is, to define the cluster as only those points within the polygon?

**Comment by aliddell** [comment]
One can split the cluster in the feature view (press 'S' when the feature window is in the foreground), and from there you can decide what to do with the other spikes (keep, delete).

**Comment by agbondy** [comment]
Sorry, I should have been more clear. What I mean is best illustrated by this example panel from the feature window. What you see here is that JRClust has identified a real unit but it has missed most of its events (i.e. grey background dots under the black cluster). What I'd like to do is redefine the unit by drawing a polygon to include all these events.  Is there a way to achieve this? I found this basic issue (JRClust assigning many of the units in a cluster to background) to be quite common.


<img width="142" alt="Capture" src="https://user-images.githubusercontent.com/7927884/58820928-ebc29d00-8601-11e9-8acc-50796c0a2b27.PNG">

**Comment by aliddell** [comment]
No, this isn't possible in JRCLUST. The only way to approximate this is by finding (by trial and error, unfortunately) which unit or units these spikes belong to and merging them with the unit you've got selected.

**Comment by agbondy** [comment]
This would be a very useful feature imo!
